corehq/toggles.py

```diff
@@ -366,3 +366,11 @@ def toggles_dict(username=None, domain=None):
     'bulk_payments',
     'Enable payment of invoices by bulk credit payments and invoice generation for wire tranfers',
 )
+
+USE_NEW_TIMEZONE_BEHAVIOR = StaticToggle(
+    'use_new_timezone_behavior',
+    ("Enable properly dealing with phone timestamps "
+     "during submission and in reports. "
+     "(Please do not set manually, "
+     "because it has to be accompanied by a migration.)")
+)
```

Comment:
Given how you're using this below you need to set the namespaces here to `[NAMESPACE_DOMAIN]`